app/controllers/api/users/areas_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class API::Users::AreasController < API::BaseController
+  def index
+    # params[:area]がnilの場合は東京都を取得
+    @users = Area.users(params[:region], params[:area] || '13')
```

Comment:
```suggestion
    tokyo_area_id = '13'
    @users = Area.users(params[:region], params[:area] || tokyo_area_id)
```
こんな感じのコードにするとコメントではなくコードで表現できるかもです。

Reply:
確かにこのように変数を作った方が分かりやすくて良いですね！
簡単な事なのに忘れてました。

Reply:
変数を使ってコードで表現するように変更しました。
https://github.com/fjordllc/bootcamp/pull/6958/commits/5a63716567ed548fe5887b6d6e6dca496ef700e5